### DISCUSSION TITLE
Ability to change the default admin `User` model using config

### DISCUSSION THREAD

**Discussion by rambhosale** [filamentphp/filament#5318] [Ideas]
I have just started using the Filament for testing so far in terms of features and usability I really liked it
I would like to have a need feature, i.e change the default model `User` that filament uses for both Users and Admins
for my use case, we have separate tables for admins and users, currently, I couldn't find any config to change this behavior, if there is an option to do so then please let me know, if there isn't any then this would be a great feature to have

**Comment by danharrin** [comment]
Define a custom guard in `config/auth.php` and point to it in the Filament config file

**Reply by rambhosale** [reply]
Thank for your suggestion, now it is working as required by my needs.

**Reply by RomkaLTU** [reply]
Filament 3 generated config file is missing a lot compared to v2, how to do it with version 3?

**Reply by RomkaLTU** [reply]
Nevermind just found that we need to add ->authGuard in AdminPanelProvider

**Comment by BouchantoufZakaria** [comment]
thank  you so mutch this was very usefull
